Formula/libgnt.rb

```diff
@@ -0,0 +1,47 @@
+class Libgnt < Formula
+  desc "NCurses toolkit for creating text-mode graphical user interfaces"
+  homepage "https://keep.imfreedom.org/libgnt/libgnt"
+  url "https://downloads.sourceforge.net/project/pidgin/libgnt/2.14.0/libgnt-2.14.0.tar.xz"
+  sha256 "6b7ea2030c9755ad9756ab4b1d3396dccaef4a712eccce34d3990042bb4b3abf"
+
+  depends_on "gtk-doc" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
+
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <gnt/gnt.h>
+
+      int main() {
+          gnt_init();
+          gnt_quit();
+
+          return 0;
+      }
+    EOS
+
+    flags = [
+      "-I#{Formula["glib"].opt_include}/glib-2.0",
+      "-I#{Formula["glib"].opt_lib}/glib-2.0/include",
+      "-I#{include}",
+      "-L#{lib}",
+      "-L#{Formula["glib"].opt_lib}",
+      "-lgnt",
+      "-lglib-2.0",
+    ]
+    system ENV.cc, *flags, "test.c", "-o", "test"
```

Comment:
```suggestion
    system ENV.cc, "test.c", *flags, "-o", "test"
```

Should be a noop on macOS, but will help on Linux a lot.